app/services/feeds/assemble_article_markdown.rb

```diff
@@ -31,6 +31,10 @@ def call
 
     private
 
+    def processed_title
```

Comment:
borrowed the term and the behavior from [`Article#processed_description`](https://github.com/forem/forem/blob/main/app/models/article.rb#L340-L346)